### ISSUE TITLE
Negative value for count rows in chbench neworder view

### ISSUE THREAD

**Issue by cirego** [MaterializeInc/materialize#4883]
I've created a view with the following source:

```
materialize=> SHOW CREATE VIEW count_neworder;
               View                |                                                            Create View                                                             
-----------------------------------+------------------------------------------------------------------------------------------------------------------------------------
 materialize.public.count_neworder | CREATE VIEW "materialize"."public"."count_neworder" AS SELECT "count"(*) AS "num_neworders" FROM "materialize"."public"."neworder"
(1 row)
````

If I query it's value, I get an apparently negative number of rows:

```
materialize=> select * from count_neworder;
 num_neworders 
---------------
       -549801
(1 row)
````

If I then create a materialized view on top of `neworder`, I get the following error when trying to select from it:

```
materialize=> create materialized view neworders as select * from neworder;
CREATE VIEW
materialize=> select * from neworders;
ERROR:  Negative multiplicity: -1 for [Int32(2304), Int32(1), Int32(1)]
```

My best guess is that something funky with Debezium is going on here.

**Comment by cirego** [comment]
With the replay tests added as part of #4833, we should add a verify step at the end to ensure that all of the counts are non-zero.

**Comment by cirego** [comment]
We now verify the contents of each view at the end of the benchmark run, marking as complete. Fixed by bdf9e7c56cf9b8a062cc5fefca1a5b7869c87819…
